s4e-backend/src/test/java/pl/cyfronet/s4e/admin/property/AdminPropertyControllerTest.java

```diff
@@ -0,0 +1,142 @@
+package pl.cyfronet.s4e.admin.property;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import pl.cyfronet.s4e.BasicTest;
+import pl.cyfronet.s4e.Constants;
+import pl.cyfronet.s4e.TestDbHelper;
+import pl.cyfronet.s4e.bean.AppUser;
+import pl.cyfronet.s4e.bean.Property;
+import pl.cyfronet.s4e.data.repository.AppUserRepository;
+import pl.cyfronet.s4e.data.repository.PropertyRepository;
+
+import static com.github.npathai.hamcrestopt.OptionalMatchers.isPresent;
+import static com.github.npathai.hamcrestopt.OptionalMatchers.isPresentAnd;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static pl.cyfronet.s4e.TestJwtUtil.jwtBearerToken;
+
+@BasicTest
+@AutoConfigureMockMvc
+class AdminPropertyControllerTest {
+    private static final String TEST_NAME = "test_name";
+    private static final String TEST_VALUE = "test value";
+
+    @Autowired
+    private AppUserRepository appUserRepository;
+
+    @Autowired
+    private PropertyRepository propertyRepository;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private TestDbHelper testDbHelper;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    private AppUser admin;
+
+    @BeforeEach
+    public void beforeEach() {
+        testDbHelper.clean();
+
+        admin = appUserRepository.save(AppUser.builder()
+                .email("admin@mail.pl")
+                .name("John")
+                .surname("Smith")
+                .password("{noop}password")
+                .enabled(true)
+                .admin(true)
+                .build());
+    }
+
```

Comment:
There are missing tests of security checks like
- propertyCreationShouldBeSeecured -> should bee authenticated, should be an admin
...

For each endpoint in controller